forge-ai/src/main/java/forge/ai/GameState.java

```diff
@@ -1327,10 +1324,7 @@ else if (info.startsWith("OnAdventure")) {
                     c.setExiledWith(c); // This seems to be the way it's set up internally. Potentially not needed here?
                     c.setExiledBy(c.getController());
                 } else if (info.startsWith("IsCommander")) {
-                    c.setCommander(true);
-                    List<Card> cmd = Lists.newArrayList(player.getCommanders());
-                    cmd.add(c);
-                    player.setCommanders(cmd);
+                    player.addCommander(c);
```

Comment:
I can no longer cast my commander when loading a state

Reply:
Should be fixed in the latest revision.